spec.bs

```diff
@@ -171,10 +171,13 @@ are:
       1. [=list/Append=] |igAd| to |interestGroup|'s |interestGroupField|.
 1. If |interestGroup|'s [=interest group/estimated size=] is greater than 50 KB, then
   [=exception/throw=] a {{TypeError}}.
+1. Let |frameOrigin| be [=this=]'s [=relevant global object=]'s `origin` property.

```

Comment:
You want to link to https://html.spec.whatwg.org/#concept-settings-object-origin via spec text like https://github.com/fedidcg/FedCM/blob/main/spec/index.bs#L420